dropbox/oauth.py

```diff
@@ -288,7 +288,6 @@ def __init__(self, consumer_key, consumer_secret=None, locale=None, token_access
         """
         Construct an instance.
 
-        Parameters
```

Comment:
Currently, this part is only rendered as parameters after _:param str locale:_  as can be seen [here](https://dropbox-sdk-python.readthedocs.io/en/latest/api/oauth.html#dropbox.oauth.DropboxOAuth2FlowNoRedirect).

Since the only difference between this and other classes where parameters are rendered correctly is the deleted part, I'd assume that that should fix it.